vars/main.yml

```diff
@@ -2,6 +2,8 @@
 openio_event_agent_sysconfig_dir: "/etc/oio/sds/{{ openio_event_agent_namespace }}"
 openio_event_agent_servicename: "oio-event-agent-{{ openio_event_agent_serviceid }}"
 
+openio_event_agent_delete_servicename: "oio-event-agent-delete-{{ openio_event_agent_serviceid }}"
```

Comment:
This contradicts usage in templates: `openio_event_agent_servicename`. Also if kept as is, add a directory check in tasks/main.yml

Reply:
I really don't like to change the service name. It has a lot of impact (log names, pid names, directory names, ...), it is not an easy change.
Please dont.